kubejobs_cost/__init__.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2020 UFCG-LSD.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from kubejobs import KubeJobProgress
+from monitor.utils.plugin import k8s
+
+import requests
+import json
+import time
+
+# Think about better metric to use
+DIVISOR = 100
```

Comment:
I don't think we need this. Shouldn't we return the error as err = rep - desired_num_of_replicas ?